changelog.md

```diff
@@ -2,15 +2,15 @@
 All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
-<br/>
+
 
 ## [Unreleased]
 
 ### Added
 - ProfileDose object built ontop of xarray. [@SimonBiggs](https://github.com/SimonBiggs).
 
-### Changed
-- **Breaking Change**: All uses of "dcm" in directory names, module names, function names, etc.
+### Breaking Changes
```

Comment:
(Referring to #66) I agree, this is probably better.

Perhaps we should consider a whole new set of change headings...